### PR TITLE
Add log in page

### PR DESCRIPTION
**Pull request by lkdfount**
- add the front end of the log in page
- make the landing page route to the log in page

### CODE REVIEW

File: travvy/src/components/LogIn.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          <br>
+          <br>
+          <label>Password: </label>
+          <input type="password" v-bind="UserForm"/> 
+        </form>
+        <br>
+        <br>
+        <button class="search"><span>Log In</span></button>
+    </div>
+</template>
```
Comment (emilynairnn) [inline travvy/src/components/LogIn.vue:19]: This looks good to me - similar to the sign up page which is great - I am happy to see we have maintained consistency!

File: travvy/src/components/LogIn.vue [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+export default {
+  name: 'LogIn',
+  props: {
+    msg: String
+  }
+}
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```
Comment (emilynairnn) [inline travvy/src/components/LogIn.vue:31]: Good to keep the style the same among various components and pages.

File: travvy/src/components/LogIn.vue [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          <label>Email: </label>
+          <input type="text" v-bind="UserForm"/> 
+          <br>
+          <br>
+          <label>Password: </label>
+          <input type="password" v-bind="UserForm"/> 
+        </form>
+        <br>
+        <br>
+        <button class="search"><span>Log In</span></button>
```
Comment (emilynairnn) [inline travvy/src/components/LogIn.vue:17]: Once the backend is created - it will be interesting to see the method behind the log in button. Good job!

**Comment by rushanniazi** [comment]
Looks good to me! Great job with the Log In page

**Comment by lucasungaretti** [comment]
LGTM

**Pull request merged**